src/main/java/io/github/externschool/planner/entity/User.java

```diff
@@ -78,29 +81,30 @@ public void setEncryptedPassword(String encryptedPassword) {
         this.encryptedPassword = encryptedPassword;
     }
 
+
     @Override
     public boolean equals(Object o) {
         if (this == o) return true;
         if (!(o instanceof User)) return false;
         User user = (User) o;
         return Objects.equals(id, user.id) &&
-                Objects.equals(phoneNumber, user.phoneNumber) &&
                 Objects.equals(email, user.email) &&
                 Objects.equals(password, user.password) &&
                 Objects.equals(encryptedPassword, user.encryptedPassword);
     }
 
+
     public UserDTO constructUser() {
-        UserDTO useDtO = new UserDTO();
-        useDtO.setEmail(this.getEmail());
-        useDtO.setPassword(this.getPassword());
-        useDtO.setPhoneNumber(this.getPhoneNumber());
-        return  useDtO;
+        UserDTO userDtO = new UserDTO();
+        userDtO.setEmail(this.getEmail());
```

Comment:
best practices say a java variable should be named with commonly used camel-case, for example <a href="https://google.github.io/styleguide/javaguide.html">Google Java Style Guide</a> , this way we get userDto or using conventional camel-case, which keeps original acronyms like userDTO. 
userDtO confronts any convention in fact.

Reply:
btw I think we should move this method (as well as reversal one from UserDTO) to a new class because it's a UserToUserDTO converter in fact. So you can keep it until being refactored